### DISCUSSION TITLE
文章发布后，无法浏览

### DISCUSSION THREAD

**Discussion by kinguler** [Licoy/wordpress-theme-puock#106] [Q&A] [answered]
你访问的资源不存在！
如何解决？

**Comment by Licoy** [comment]
@kinguler 重新保存一下固定链接试试

**Comment by kinguler** [comment, marked as answer]
我装了个 插件 解决了 是链接用中文问题




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年1月15日(星期六) 晚上8:00
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Licoy/wordpress-theme-puock] 文章发布后，无法浏览 (Discussion #106)





 
@kinguler 重新保存一下固定链接试试
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by Licoy** [reply]
好的
